scripts/split-paired-reads.py

```diff
@@ -92,11 +92,15 @@ def main():
 
     infile = args.infile
 
-    check_input_files(infile, args.force)
```

Comment:
In general, you should avoid "noise" in the change - here you moved check_input_files down by a line, creating a change for no reason.  It's OK, but please try to avoid in the future. tnx!